### PR TITLE
Add real time notification test app 

### PR DESCRIPTION
**Pull request by marche0133**
To follow the iOS sdk, we put this test app into SDK repo. Originally this app is for internal testing of real time notification (signaling) purpose. We may update this app furthermore to make it into a demo app that demo chat SDK functionalities in future.

### CODE REVIEW

**Review by vcolin7 — APPROVED** [review]
In general this looks pretty good to me, I haven't had the time to download and test it but I'm approving it to unblock for the time being. Left a couple questions and suggestions I would like you to consider before merging :)

File: samples/sample-chat-app/build.gradle
Comment (vcolin7) [reply samples/sample-chat-app/build.gradle:11]: I don't think we currently have guidance for adding samples in a specific package, these are mere suggestions from the top of my head.

File: samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java
Comment (vcolin7) [inline samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java:47]: Is this a valid token? We do not want to commit credentials to our repositories at any point in time.

Comment (marche0133) [reply samples/sample-chat-app/src/main/java/com/azure/android/communication/chat/sampleapp/MainActivity.java:47]: Token expires in 24 hours, but this one I accidentally commit, let me revert it.

**Pull request merged**
